sunpy/net/dataretriever/sources/lyra.py

```diff
@@ -32,8 +32,18 @@ class LYRAClient(GenericClient):
     2016-01-02 00:00:00.000 2016-01-02 23:59:59.999       LYRA ...      ESA     2
     2016-01-02 00:00:00.000 2016-01-02 23:59:59.999       LYRA ...      ESA     3
     <BLANKLINE>
+    3 Results from the VSOClient:
+    Source: http://vso.stanford.edu/cgi-bin/search
+    Total estimated size: 2.914 Gbyte
+    <BLANKLINE>
+           Start Time       ...          Info Required
+                            ...
+    ----------------------- ... --------------------------------
+    2016-01-01 09:41:00.000 ... ...
```

Comment:
The info required values seemed to change on each query so I ignored them.

Reply:
We shouldn't be printing that column at all, see my comment on #6887